builder/processor/src/main/java/io/helidon/builder/processor/Types.java

```diff
@@ -81,6 +81,8 @@ final class Types {
     static final TypeName PROTOTYPE_CUSTOM_METHOD_TYPE = TypeName.create(PROTOTYPE_CUSTOM_METHOD);
     static final TypeName BUILDER_CUSTOM_METHOD_TYPE = TypeName.create(BUILDER_CUSTOM_METHOD);
     static final TypeName DESCRIPTION_TYPE = TypeName.create(DESCRIPTION);
+    static final TypeName DEPRECATED_TYPE = TypeName.create(Deprecated.class);
+    static final TypeName OPTION_DEPRECATED_TYPE = TypeName.create("io.helidon.builder.api.Option.Deprecated");
```

Comment:
why not use the class version of this - same for `Option.Type` at line 103.

Reply:
I am not sure I understand. Deprecated is on the classpath, Option.Deprecated is not.